pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSources.java

```diff
@@ -216,10 +224,6 @@ void runCmd() throws Exception {
             }
             print("Updated successfully");
         }
-
-        protected void validateSourceConfigs(SourceConfig sourceConfig) {
```

Comment:
Same as my comment about removing this for sinks

Reply:
done